### PR TITLE
Add RDS and Test

### PR DESCRIPTION
**Pull request by shoukoo**
I only added a test for RDS because we might store alerts in the tag for the other types.

### CODE REVIEW

File: rds_test.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	"depends_on": [
+		"aws_db_subnet_group.postgres",
+		"aws_security_group.postgres",
+		"local.terraform"
+	],
+	"primary": {
+		"id": "au-identity-equifax",
+		"attributes": {
+			"tags.Name": "au-identity-equifax",
+			"tags.alert": "below 10 pulse in 30: above 2000 pulse in 30",
```
Comment (alexlance) [inline rds_test.go]: There's a colon character separating the alerts. It should either be a semi-colon or a comma. I think maybe semi-colon is good

File: rds_test.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
```
Comment (alexlance) [inline rds_test.go]: same here

File: rds_test.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	alerts, err := rds.Parse(rds.AlertTag)
+	if err != nil && err.Error() != "strconv.Atoi: parsing \"as\": invalid syntax" {
+		t.Errorf("Invalid Error type")
+	}
+
+	//Invalid character count
+	rds.Type = "alert"
+	rds.AlertTag = "below 10 pulse"
+	rds.Name = "identity-uk"
+	if err.Error() != fmt.Sprintf("%v alert condition needs to contain 5 characters", rds.AlertTag) {
```
Comment (alexlance) [inline rds_test.go]: I don't quite understand the five characters thing.
I think there are basically three modes:
the alert tag is not set
the alert tag is set, but it is an empty string
the alert tag is set, and has one or more alerts defined in it

Comment (shoukoo) [reply rds_test.go]: I am just guessing that a valid alert condition needs to contain the exact 5 alphabets? "Below 30 pulse in 30"
"Below 30 cpu in 120"
 
Terra-map should error out if an alert condition without a duration - "Below 30 pulse", right?

File: rds_test.go (hunk 1 above)
Comment (shoukoo) [reply rds_test.go]: `Tag values may only contain unicode letters, digits, whitespace, or one of these symbols: _ . : / = + - @`

I forgot to let you know but Tag doesn't accept comma or semi-colon.

File: rds.go [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+		if s.Type == "alert" {
+			m.Details.Alert = rule
+		} else if s.Type == "warn" {
+			m.Details.Warn = rule
+		}
+		m.Details.ID = s.Name
+		m.Details.Duration = duration
+		return m, nil
+	} else {
+		return nil, fmt.Errorf("%v alert condition needs to contain 5 characters", tag)
```
Comment (alexlance) [inline rds.go]: I think the word "characters" needs to be the word "words"

**Pull request merged**